### PR TITLE
FIS-1: Create card component

### PR DESCRIPTION
**Pull request by HarelSon**

### CODE REVIEW

File: src/pages/index.astro [hunk 1]
```diff
@@ -1,7 +1,27 @@
 ---
 import Layout from "../layouts/Layout.astro";
+import Card from "../components/Card.astro";
 ---
 
 <Layout>
-  <h1 class="font-sans text-4xl">nunito</h1>
+  <div class="flex gap-8">
+    <Card
+      class="bg-[#9AD3CC]"
```
Comment (jardix22) [inline src/pages/index.astro]: Please use a tailwind variable color like `bg-yellow`. To achieve this goal you need to add a custom color in the `tailwind.config.cjs`.

File: src/pages/index.astro [hunk 2]
```diff
@@ -1,7 +1,27 @@
...
+      icon="check-circle"
+      title="Save taxes"
+      parrafo="PFRDA appinted POP (Point of Presence) registretion number 1614858 and were authorized to provide NPS services."
+    />
+    <Card
+      class="bg-[#EEEDEA]"
```
Comment (jardix22) [inline src/pages/index.astro]: Please do the same thing that I said above.

File: src/pages/index.astro [hunk 3]
```diff
@@ -1,7 +1,27 @@
...
+      icon="percent"
+      title="Regulated & Transparent"
+      parrafo="Under Section 80CCD, save upto 15,480 in taxes."
+    />
+    <Card
+      class="bg-[#E8CAA2]"
```
Comment (jardix22) [inline src/pages/index.astro]: Please do the same thing that I said above.

**Review by jardix22 — CHANGES_REQUESTED** [review]

File: src/pages/index.astro (hunk 1 above)
Comment (jardix22) [reply src/pages/index.astro]: Here you can find some documentation [Adding additional colors](https://tailwindcss.com/docs/customizing-colors#adding-additional-colors).

Comment (HarelSon) [reply src/pages/index.astro]: I replaced arbitrary values

**Review by jardix22 — APPROVED** [review]

**Pull request merged**